src/pages/index.js

```diff
@@ -69,8 +145,24 @@ const popupAddForm = new PopupWithForm('#add', handleProfileAddSubmit);
 popupAddForm.setEventListeners();
 //
 
+//PopupWithForm - Profile
+const popupProfileImage = new PopupWithForm('#profile', handleProfileImageSubmit);
+popupProfileImage.setEventListeners();
+//
+
+//Popup - Delete Button
+const popupDeleteButton = new Popup('#delete'); //was creating with PopupWithForm but was giving it multiple event listeners for the submission
```

Comment:
Class `Popup` should not be used in `index.js` because it's only for **extending**. Please, make `PopupWithConfirmation` according to the checklist